sphinx/source/conf.py

```diff
@@ -85,17 +85,34 @@
             'deephaven.dataDir': os.path.realpath(workspace),
         }
 
+jvm_options = {
+    '-XX:InitialRAMPercentage=25.0',
+    '-XX:MinRAMPercentage=70.0',
+    '-XX:MaxRAMPercentage=80.0',
+
+    # Allow netty to (reflectively) access java.nio.Buffer fields
```

Comment:
Is this valid? Only netty? Does sphinx still use netty?

Reply:
I think the netty part is too specific. It looks like it's also required by arrow, and potentially other low-level components. I've removed the explicit reference to "netty".